### PR TITLE
add an option to save checkpoints at a specific epoch

### PR DESCRIPTION
**Pull request by zydou**
The best checkpoints rarely occurred in the first few epochs. So adding this option will reduce the trainning time.
For example:
```
python examples/triplet_loss.py -d market1501 -a resnet50 --combine-trainval --logs-dir examples/logs/triplet-loss/market1501-resnet50
```
and
```
python examples/triplet_loss.py --start_save 130 -d market1501 -a resnet50 --combine-trainval --logs-dir examples/logs/triplet-loss/market1501-resnet50
```
gives the same results. But the latter is faster.

### CODE REVIEW

File: examples/oim_loss.py [hunk 1]
```diff
@@ -207,6 +209,8 @@ def adjust_lr(epoch):
     parser.add_argument('--evaluate', action='store_true',
                         help="evaluation only")
     parser.add_argument('--epochs', type=int, default=50)
+    parser.add_argument('--start_save', type=int, default=0,
+                        help="save checkpoints at specific epoch")
```
Comment (Cysu) [inline examples/oim_loss.py]: Maybe it's better to say "start saving checkpoints after specific epoch"?

Comment (zydou) [reply examples/oim_loss.py]: Yes！It's more clear！

Comment (Cysu) [reply examples/oim_loss.py]: Could you please change the help text when available and push the commit? This PR will automatically track the changes and I can then merge it directly. Thanks.

Comment (zydou) [reply examples/oim_loss.py]: OK.I'll do it tomorrow.

Comment (Cysu) [reply examples/oim_loss.py]: Sure. No problem. Thanks a lot!

Comment (zydou) [reply examples/oim_loss.py]: Done!

**Pull request merged**